### PR TITLE
Refinement to migration docs after testing with customers

### PR DESCRIPTION
**Pull request by cadenzajon**

### CODE REVIEW

File: xml/art_migration.xml [hunk 1]
```diff
@@ -840,48 +850,51 @@ mig-cp1-neut-m1-mgmt</screen>
...
+<prompt>(new-deployer)> </prompt>rm -rf .ansible-fact-cache
 <prompt>(new-deployer)> </prompt>sudo mkdir -p /media/cdrom
-<prompt>(new-deployer)> </prompt>echo /home/stack/sles12sp3.iso /media/cdrom iso9660 loop 0 0 \
-  | sudo tee -a /etc/fstab
-<prompt>(new-deployer)> </prompt>sudo mount -a
+<prompt>(new-deployer)> </prompt>sudo mount /home/stack/sles12sp3.iso /media/cdrom
+<prompt>(new-deployer)> </prompt>sudo mount HPE-HELION-OPENSTACK-5-TO-8-MIGRATION-SCP-x86_64-991.iso hos-5.scp
 <prompt>(new deployer)> </prompt>tar -xf hos-5.scp/hos/hos-5.1.0-*.tar
 <prompt>(new deployer)> </prompt>cd ~/hos-5.1.0
 <prompt>(new deployer)> </prompt>./hos-init.bash
```
Comment (KeithMnemonic) [inline xml/art_migration.xml]: this is where we see the error it the new deployer is not standalone but hosted on a controller. we need to verify if this occurs all of the time.

TASK: [KEY-API | status | Register keystone status] ***************************
fatal: [localhost] => One or more undefined variables: 'host' is undefined

FATAL: all hosts have already failed -- aborting

a cp/readty resolves it, maybe a refresh facts as well

File: xml/art_migration.xml [hunk 2]
```diff
@@ -576,24 +584,14 @@ mig-cp1-neut-m1-mgmt</screen>
...
-       seed node during installation.
-      </para>
-      <para>
-       If your compute nodes are kept up to date using the &suse; Updates
-       repository via an SMT server, you should see a repository for this
-       purpose defined on your compute nodes. For proper behavior during the
-       upgrade, your repository names should be <systemitem>SLES12-SP3-Pool</systemitem>
-       and <systemitem>SLES12-SP3-Updates</systemitem>. If necessary, you can
-       rename any existing repository to match these names by running:
+       On each SLES compute node, remove all the repositories except for
```
Comment (KeithMnemonic) [inline xml/art_migration.xml]: The title above this should be Update repository references of Compute Node**s**

Comment (KeithMnemonic) [reply xml/art_migration.xml]: not sure Node is capitalized, if it is not there are other places as well

File: xml/art_migration.xml [hunk 3]
```diff
@@ -1348,7 +1397,7 @@ ansible-playbook -i hosts/verb_hosts _osconfig-setup-repos.yml</screen>
      </listitem>
      <listitem>
       <para>
-       HPE-Helion-OpenStack-8-Pool
+       HPE-Helion-penStack-8-Pool
```
Comment (KeithMnemonic) [inline xml/art_migration.xml]: mising O

File: xml/art_migration.xml [hunk 4]
```diff
@@ -840,48 +850,51 @@ mig-cp1-neut-m1-mgmt</screen>
...
        Connect to the new deployer and initialize it for use:
       </para>
 <screen><prompt>(new-deployer)> </prompt>cd ~
+<prompt>(new-deployer)> </prompt>rm -rf .ansible-fact-cache
 <prompt>(new-deployer)> </prompt>sudo mkdir -p /media/cdrom
-<prompt>(new-deployer)> </prompt>echo /home/stack/sles12sp3.iso /media/cdrom iso9660 loop 0 0 \
-  | sudo tee -a /etc/fstab
-<prompt>(new-deployer)> </prompt>sudo mount -a
+<prompt>(new-deployer)> </prompt>sudo mount /home/stack/sles12sp3.iso /media/cdrom
+<prompt>(new-deployer)> </prompt>sudo mount HPE-HELION-OPENSTACK-5-TO-8-MIGRATION-SCP-x86_64-991.iso hos-5.scp
```
[the remark this answers is not in the document]
Comment (cadenzajon) [reply xml/art_migration.xml]: Actually we are using full iso name everywhere now.

File: xml/art_migration.xml (hunk 1 above)
Comment (cadenzajon) [reply xml/art_migration.xml]: This code I added should resolve the issue:
rm -rf .ansible-fact-cache

File: xml/art_migration.xml [hunk 5]
```diff
@@ -747,14 +735,36 @@ mig-cp1-neut-m1-mgmt</screen>
...
+       In a mid-scale cloud topology, some data files may have been
+       stored on the non-primary drive(s) in the system. If this was the case,
+       the <filename>wipe_disk.yml</filename> playbook should be used (after the
+       nodes have been rebuilt) to purge the old data. For example:
+      </para>
+<screen><prompt>(seed)> </prompt>cd ~/scratch/ansible/next/hos/ansible
+<prompt>(seed)> </prompt>ansible-playbook -i hosts/verb_hosts hlm-refresh-facts.yml
+<prompt>(seed)> </prompt>ansible-playbook -i hosts/verb_hosts wipe_disks.yml \
+  --limit dbmq-m1,dbmq-m2,dbmq-m3</screen>
+      <para>
```
[the remark this answers is not in the document]
Comment (cadenzajon) [reply xml/art_migration.xml]: Fixed

**Comment by cadenzajon** [comment]
Latest patches incorporates all feedback from Keth

**Pull request merged**